src/main/java/com/ibm/g11n/pipeline/client/ServiceClient.java

```diff
@@ -626,4 +629,137 @@ public abstract TranslationConfigData getTranslationConfig(String sourceLanguage
      */
     public abstract void deleteTranslationConfig(String sourceLanguage, String targetLanguage)
            throws ServiceException;
+
+
+    //
+    // {serviceInstanceId}/v2/trs APIs
+    //
+
+    /**
+     * Returns a map containing <code>TranslationRequestData</code> indexed by translation
+     * request IDs.
+     * 
+     * @return A map containing <code>TranslationRequestData</code> indexed by translation.
+     * @throws ServiceException when the operation failed.
+     */
+    public abstract Map<String, TranslationRequestData> getTranslationRequests() throws ServiceException;
+
+    /**
+     * Returns the translation request specified by the translation request ID.
+     * 
+     * @param trId  The translation request ID.
+     * @return  The translation request data.
+     * @throws ServiceException when the operation failed.
+     */
+    public abstract TranslationRequestData getTranslationRequest(String trId)
+            throws ServiceException;
+
+    /**
+     * Creates a new translation request.
+     * 
+     * @param newTranslationRequestData The new translation request.
+     * @return  The translation request created by this operation.
+     * @throws ServiceException when the operation failed.
+     */
+    public abstract TranslationRequestData createTranslationRequest(NewTranslationRequestData newTranslationRequestData)
+            throws ServiceException;
+
+    /**
+     * Updates the translation request.
```

Comment:
"if the TR is in SUBMIT state, it willbe submitted" ?
or have a ",boolean draft=false)"  where the TR will be submitted unless you set it to true?